t/e2e/api/api_test.go

```diff
@@ -77,19 +77,19 @@ func (s *APITestSuite) TestRaceConditions() {
 	var funcsToTest = []func(*params.NodeConfig){
 		func(config *params.NodeConfig) {
 			log.Info("StartNodeAsync()")
```

Comment:
What do we want to do with these log entries? `StartNodeAsync()` no longer exists, and we're making 2 log entries one after another, which seems redundant (maybe there's a reason why one is using `log.Info` and the other `s.T().Logf`).

Reply:
@PombeirP what do you think about a single log entry just before the runAsync call like:
```
s.T().Logf("Async call to StartNode() for network: %d", config.NetworkID)
api.RunAsync(func() error { return s.backend.StartNode(config) })
```

Reply:
@adriacidre sounds good :+1: